cypress/integration/app.spec.js

```diff
@@ -40,11 +40,87 @@ describe("Basic e2e", function () {
     // Hacky way to get past RHF watch -method problem that doesn't allow cypress to get updated value for file
     cy.get("form").submit()
 
-    // Saved objects list should have newly added item
+    // Saved objects list should have newly added item from Study object
     cy.get(".MuiListItem-container", { timeout: 10000 }).should("have.length", 2)
 
-    // // Navigate to summary and publish
+    // Fill an Analysis form and submit object
+    cy.get("div[role=button]").contains("Analysis").click()
+    cy.get("div[role=button]").contains("Fill Form").click()
```

Comment:
```suggestion
    cy.get("div[role=button]").contains("Fill Form").click({force: true})
```

I am getting the error below for some reason:

![image](https://user-images.githubusercontent.com/47524/104129744-324bb200-5376-11eb-8cab-a58262bece9e.png)

Reply:
There seems to be latency in Cypress and for some reason it sees the list as hidden too long. One option would be to add longer timeout as parameter for `get` method.

As an side note:
I've seen Cypress to pass if ran again in headless browser. Initial run can expose some problems with Cypress latency issues but these need to be addressed since these issues may exist with automated e2e tests on GH.

Reply:
Could you try with my latest fix

Reply:
works now. thx!